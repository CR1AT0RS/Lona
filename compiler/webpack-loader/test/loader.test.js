const compiler = require('./compiler')

test('Should compile a component', () => {
  return compiler(
    '../../../examples/test/components/NestedButtons.component'
  ).then(stats => {
    const output = stats
      .toJson()
      .modules.find(x =>
        x.reasons.some(
          y =>
            y.userRequest ===
            './../../../examples/test/components/NestedButtons.component'
        )
      )

    expect(output.modules.map(x => x.source)).toMatchSnapshot()
  })
})
