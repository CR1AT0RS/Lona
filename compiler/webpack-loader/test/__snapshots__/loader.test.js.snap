// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should compile a component 1`] = `
Array [
  "function _typeof(obj) { if (typeof Symbol === \\"function\\" && typeof Symbol.iterator === \\"symbol\\") { _typeof = function _typeof(obj) { return typeof obj; }; } else { _typeof = function _typeof(obj) { return obj && typeof Symbol === \\"function\\" && obj.constructor === Symbol && obj !== Symbol.prototype ? \\"symbol\\" : typeof obj; }; } return _typeof(obj); }

function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError(\\"Cannot call a class as a function\\"); } }

function _defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if (\\"value\\" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } }

function _createClass(Constructor, protoProps, staticProps) { if (protoProps) _defineProperties(Constructor.prototype, protoProps); if (staticProps) _defineProperties(Constructor, staticProps); return Constructor; }

function _possibleConstructorReturn(self, call) { if (call && (_typeof(call) === \\"object\\" || typeof call === \\"function\\")) { return call; } return _assertThisInitialized(self); }

function _assertThisInitialized(self) { if (self === void 0) { throw new ReferenceError(\\"this hasn't been initialised - super() hasn't been called\\"); } return self; }

function _getPrototypeOf(o) { _getPrototypeOf = Object.setPrototypeOf ? Object.getPrototypeOf : function _getPrototypeOf(o) { return o.__proto__ || Object.getPrototypeOf(o); }; return _getPrototypeOf(o); }

function _inherits(subClass, superClass) { if (typeof superClass !== \\"function\\" && superClass !== null) { throw new TypeError(\\"Super expression must either be null or a function\\"); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, writable: true, configurable: true } }); if (superClass) _setPrototypeOf(subClass, superClass); }

function _setPrototypeOf(o, p) { _setPrototypeOf = Object.setPrototypeOf || function _setPrototypeOf(o, p) { o.__proto__ = p; return o; }; return _setPrototypeOf(o, p); }

import React from \\"react\\";
import colors from \\"../colors.js!=!/Users/mathieudutour/Projects/Lona/compiler/webpack-loader/lib/loader.js!../colors.json?__forceLona=1\\";
import shadows from \\"../shadows.js!=!/Users/mathieudutour/Projects/Lona/compiler/webpack-loader/lib/loader.js!../shadows.json?__forceLona=1\\";
import textStyles from \\"../textStyles.js!=!/Users/mathieudutour/Projects/Lona/compiler/webpack-loader/lib/loader.js!../textStyles.json?__forceLona=1\\";
import Button from \\"../interactivity/Button.component\\";

var NestedButtons =
/*#__PURE__*/
function (_React$Component) {
  _inherits(NestedButtons, _React$Component);

  function NestedButtons() {
    _classCallCheck(this, NestedButtons);

    return _possibleConstructorReturn(this, _getPrototypeOf(NestedButtons).apply(this, arguments));
  }

  _createClass(NestedButtons, [{
    key: \\"render\\",
    value: function render() {
      return React.createElement(\\"div\\", {
        style: styles.view
      }, React.createElement(\\"div\\", {
        style: styles.button
      }, React.createElement(Button, {
        label: \\"Button 1\\"
      })), React.createElement(\\"div\\", {
        style: styles.view1
      }), React.createElement(\\"div\\", {
        style: styles.button2
      }, React.createElement(Button, {
        label: \\"Button 2\\"
      })));
    }
  }]);

  return NestedButtons;
}(React.Component);

export { NestedButtons as default };
;
var styles = {
  view: {
    alignItems: \\"flex-start\\",
    display: \\"flex\\",
    flex: \\"1 1 0%\\",
    flexDirection: \\"column\\",
    justifyContent: \\"flex-start\\",
    paddingTop: \\"24px\\",
    paddingRight: \\"24px\\",
    paddingBottom: \\"24px\\",
    paddingLeft: \\"24px\\"
  },
  button: {
    alignItems: \\"flex-start\\",
    alignSelf: \\"stretch\\",
    display: \\"flex\\",
    flex: \\"1 1 auto\\",
    flexDirection: \\"row\\",
    justifyContent: \\"flex-start\\"
  },
  view1: {
    alignItems: \\"flex-start\\",
    alignSelf: \\"stretch\\",
    display: \\"flex\\",
    flexDirection: \\"column\\",
    justifyContent: \\"flex-start\\",
    height: \\"8px\\"
  },
  button2: {
    alignItems: \\"flex-start\\",
    alignSelf: \\"stretch\\",
    display: \\"flex\\",
    flex: \\"1 1 auto\\",
    flexDirection: \\"row\\",
    justifyContent: \\"flex-start\\"
  }
};",
  "export default {
  red50: \\"#ffebee\\", // Test comment
  red100: \\"#ffcdd2\\",
  red200: \\"#ef9a9a\\",
  red300: \\"#e57373\\",
  red400: \\"#ef5350\\",
  red500: \\"#f44336\\",
  red600: \\"#e53935\\",
  red700: \\"#d32f2f\\",
  red800: \\"#c62828\\",
  red900: \\"#b71c1c\\",
  reda100: \\"#ff8a80\\",
  reda200: \\"#ff5252\\",
  reda400: \\"#ff1744\\",
  reda700: \\"#d50000\\",
  pink50: \\"#fce4ec\\",
  pink100: \\"#f8bbd0\\",
  pink200: \\"#f48fb1\\",
  pink300: \\"#f06292\\",
  pink400: \\"#ec407a\\",
  pink500: \\"#e91e63\\",
  pink600: \\"#d81b60\\",
  pink700: \\"#c2185b\\",
  pink800: \\"#ad1457\\",
  pink900: \\"#880e4f\\",
  pinka100: \\"#ff80ab\\",
  pinka200: \\"#ff4081\\",
  pinka400: \\"#f50057\\",
  pinka700: \\"#c51162\\",
  purple50: \\"#f3e5f5\\",
  purple100: \\"#e1bee7\\",
  purple200: \\"#ce93d8\\",
  purple300: \\"#ba68c8\\",
  purple400: \\"#ab47bc\\",
  purple500: \\"#9c27b0\\",
  purple600: \\"#8e24aa\\",
  purple700: \\"#7b1fa2\\",
  purple800: \\"#6a1b9a\\",
  purple900: \\"#4a148c\\",
  purplea100: \\"#ea80fc\\",
  purplea200: \\"#e040fb\\",
  purplea400: \\"#d500f9\\",
  purplea700: \\"#aa00ff\\",
  deeppurple50: \\"#ede7f6\\",
  deeppurple100: \\"#d1c4e9\\",
  deeppurple200: \\"#b39ddb\\",
  deeppurple300: \\"#9575cd\\",
  deeppurple400: \\"#7e57c2\\",
  deeppurple500: \\"#673ab7\\",
  deeppurple600: \\"#5e35b1\\",
  deeppurple700: \\"#512da8\\",
  deeppurple800: \\"#4527a0\\",
  deeppurple900: \\"#311b92\\",
  deeppurplea100: \\"#b388ff\\",
  deeppurplea200: \\"#7c4dff\\",
  deeppurplea400: \\"#651fff\\",
  deeppurplea700: \\"#6200ea\\",
  indigo50: \\"#e8eaf6\\",
  indigo100: \\"#c5cae9\\",
  indigo200: \\"#9fa8da\\",
  indigo300: \\"#7986cb\\",
  indigo400: \\"#5c6bc0\\",
  indigo500: \\"#3f51b5\\",
  indigo600: \\"#3949ab\\",
  indigo700: \\"#303f9f\\",
  indigo800: \\"#283593\\",
  indigo900: \\"#1a237e\\",
  indigoa100: \\"#8c9eff\\",
  indigoa200: \\"#536dfe\\",
  indigoa400: \\"#3d5afe\\",
  indigoa700: \\"#304ffe\\",
  blue50: \\"#e3f2fd\\",
  blue100: \\"#bbdefb\\",
  blue200: \\"#90caf9\\",
  blue300: \\"#64b5f6\\",
  blue400: \\"#42a5f5\\",
  blue500: \\"#2196f3\\",
  blue600: \\"#1e88e5\\",
  blue700: \\"#1976d2\\",
  blue800: \\"#1565c0\\",
  blue900: \\"#0d47a1\\",
  bluea100: \\"#82b1ff\\",
  bluea200: \\"#448aff\\",
  bluea400: \\"#2979ff\\",
  bluea700: \\"#2962ff\\",
  lightblue50: \\"#e1f5fe\\",
  lightblue100: \\"#b3e5fc\\",
  lightblue200: \\"#81d4fa\\",
  lightblue300: \\"#4fc3f7\\",
  lightblue400: \\"#29b6f6\\",
  lightblue500: \\"#03a9f4\\",
  lightblue600: \\"#039be5\\",
  lightblue700: \\"#0288d1\\",
  lightblue800: \\"#0277bd\\",
  lightblue900: \\"#01579b\\",
  lightbluea100: \\"#80d8ff\\",
  lightbluea200: \\"#40c4ff\\",
  lightbluea400: \\"#00b0ff\\",
  lightbluea700: \\"#0091ea\\",
  cyan50: \\"#e0f7fa\\",
  cyan100: \\"#b2ebf2\\",
  cyan200: \\"#80deea\\",
  cyan300: \\"#4dd0e1\\",
  cyan400: \\"#26c6da\\",
  cyan500: \\"#00bcd4\\",
  cyan600: \\"#00acc1\\",
  cyan700: \\"#0097a7\\",
  cyan800: \\"#00838f\\",
  cyan900: \\"#006064\\",
  cyana100: \\"#84ffff\\",
  cyana200: \\"#18ffff\\",
  cyana400: \\"#00e5ff\\",
  cyana700: \\"#00b8d4\\",
  teal50: \\"#e0f2f1\\",
  teal100: \\"#b2dfdb\\",
  teal200: \\"#80cbc4\\",
  teal300: \\"#4db6ac\\",
  teal400: \\"#26a69a\\",
  teal500: \\"#009688\\",
  teal600: \\"#00897b\\",
  teal700: \\"#00796b\\",
  teal800: \\"#00695c\\",
  teal900: \\"#004d40\\",
  teala100: \\"#a7ffeb\\",
  teala200: \\"#64ffda\\",
  teala400: \\"#1de9b6\\",
  teala700: \\"#00bfa5\\",
  green50: \\"#e8f5e9\\",
  green100: \\"#c8e6c9\\",
  green200: \\"#a5d6a7\\",
  green300: \\"#81c784\\",
  green400: \\"#66bb6a\\",
  green500: \\"#4caf50\\",
  green600: \\"#43a047\\",
  green700: \\"#388e3c\\",
  green800: \\"#2e7d32\\",
  green900: \\"#1b5e20\\",
  greena100: \\"#b9f6ca\\",
  greena200: \\"#69f0ae\\",
  greena400: \\"#00e676\\",
  greena700: \\"#00c853\\",
  lightgreen50: \\"#f1f8e9\\",
  lightgreen100: \\"#dcedc8\\",
  lightgreen200: \\"#c5e1a5\\",
  lightgreen300: \\"#aed581\\",
  lightgreen400: \\"#9ccc65\\",
  lightgreen500: \\"#8bc34a\\",
  lightgreen600: \\"#7cb342\\",
  lightgreen700: \\"#689f38\\",
  lightgreen800: \\"#558b2f\\",
  lightgreen900: \\"#33691e\\",
  lightgreena100: \\"#ccff90\\",
  lightgreena200: \\"#b2ff59\\",
  lightgreena400: \\"#76ff03\\",
  lightgreena700: \\"#64dd17\\",
  lime50: \\"#f9fbe7\\",
  lime100: \\"#f0f4c3\\",
  lime200: \\"#e6ee9c\\",
  lime300: \\"#dce775\\",
  lime400: \\"#d4e157\\",
  lime500: \\"#cddc39\\",
  lime600: \\"#c0ca33\\",
  lime700: \\"#afb42b\\",
  lime800: \\"#9e9d24\\",
  lime900: \\"#827717\\",
  limea100: \\"#f4ff81\\",
  limea200: \\"#eeff41\\",
  limea400: \\"#c6ff00\\",
  limea700: \\"#aeea00\\",
  yellow50: \\"#fffde7\\",
  yellow100: \\"#fff9c4\\",
  yellow200: \\"#fff59d\\",
  yellow300: \\"#fff176\\",
  yellow400: \\"#ffee58\\",
  yellow500: \\"#ffeb3b\\",
  yellow600: \\"#fdd835\\",
  yellow700: \\"#fbc02d\\",
  yellow800: \\"#f9a825\\",
  yellow900: \\"#f57f17\\",
  yellowa100: \\"#ffff8d\\",
  yellowa200: \\"#ffff00\\",
  yellowa400: \\"#ffea00\\",
  yellowa700: \\"#ffd600\\",
  amber50: \\"#fff8e1\\",
  amber100: \\"#ffecb3\\",
  amber200: \\"#ffe082\\",
  amber300: \\"#ffd54f\\",
  amber400: \\"#ffca28\\",
  amber500: \\"#ffc107\\",
  amber600: \\"#ffb300\\",
  amber700: \\"#ffa000\\",
  amber800: \\"#ff8f00\\",
  amber900: \\"#ff6f00\\",
  ambera100: \\"#ffe57f\\",
  ambera200: \\"#ffd740\\",
  ambera400: \\"#ffc400\\",
  ambera700: \\"#ffab00\\",
  orange50: \\"#fff3e0\\",
  orange100: \\"#ffe0b2\\",
  orange200: \\"#ffcc80\\",
  orange300: \\"#ffb74d\\",
  orange400: \\"#ffa726\\",
  orange500: \\"#ff9800\\",
  orange600: \\"#fb8c00\\",
  orange700: \\"#f57c00\\",
  orange800: \\"#ef6c00\\",
  orange900: \\"#e65100\\",
  orangea100: \\"#ffd180\\",
  orangea200: \\"#ffab40\\",
  orangea400: \\"#ff9100\\",
  orangea700: \\"#ff6d00\\",
  deeporange50: \\"#fbe9e7\\",
  deeporange100: \\"#ffccbc\\",
  deeporange200: \\"#ffab91\\",
  deeporange300: \\"#ff8a65\\",
  deeporange400: \\"#ff7043\\",
  deeporange500: \\"#ff5722\\",
  deeporange600: \\"#f4511e\\",
  deeporange700: \\"#e64a19\\",
  deeporange800: \\"#d84315\\",
  deeporange900: \\"#bf360c\\",
  deeporangea100: \\"#ff9e80\\",
  deeporangea200: \\"#ff6e40\\",
  deeporangea400: \\"#ff3d00\\",
  deeporangea700: \\"#dd2c00\\",
  brown50: \\"#efebe9\\",
  brown100: \\"#d7ccc8\\",
  brown200: \\"#bcaaa4\\",
  brown300: \\"#a1887f\\",
  brown400: \\"#8d6e63\\",
  brown500: \\"#795548\\",
  brown600: \\"#6d4c41\\",
  brown700: \\"#5d4037\\",
  brown800: \\"#4e342e\\",
  brown900: \\"#3e2723\\",
  grey50: \\"#fafafa\\",
  grey100: \\"#f5f5f5\\",
  grey200: \\"#eeeeee\\",
  grey300: \\"#e0e0e0\\",
  grey400: \\"#bdbdbd\\",
  grey500: \\"#9e9e9e\\",
  grey600: \\"#757575\\",
  grey700: \\"#616161\\",
  grey800: \\"#424242\\",
  grey900: \\"#212121\\",
  bluegrey50: \\"#eceff1\\",
  bluegrey100: \\"#cfd8dc\\",
  bluegrey200: \\"#b0bec5\\",
  bluegrey300: \\"#90a4ae\\",
  bluegrey400: \\"#78909c\\",
  bluegrey500: \\"#607d8b\\",
  bluegrey600: \\"#546e7a\\",
  bluegrey700: \\"#455a64\\",
  bluegrey800: \\"#37474f\\",
  bluegrey900: \\"#263238\\"
};

",
  "import colors from \\"./colors.js!=!/Users/mathieudutour/Projects/Lona/compiler/webpack-loader/lib/loader.js!./colors.json?__forceLona=1\\"

export default {
  elevation1: { filter: \\"drop-shadow(0px 1px 2px rgba(0,0,0,0.5))\\" },
  elevation2: { filter: \\"drop-shadow(0px 2px 4px rgba(0,0,0,0.5))\\" },
  elevation3: { filter: \\"drop-shadow(0px 3px 9px \\" + colors.grey900 + \\")\\" }
};

",
  "import colors from \\"./colors.js!=!/Users/mathieudutour/Projects/Lona/compiler/webpack-loader/lib/loader.js!./colors.json?__forceLona=1\\"

export default {
  display4: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"300\\",
    fontSize: \\"112px\\",
    lineHeight: \\"120px\\",
    color: colors.grey900
  },
  display3: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"400\\",
    fontSize: \\"56px\\",
    lineHeight: \\"60px\\",
    color: colors.grey900
  },
  display2: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"400\\",
    fontSize: \\"45px\\",
    lineHeight: \\"48px\\",
    color: colors.grey900
  },
  display1: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"400\\",
    fontSize: \\"34px\\",
    lineHeight: \\"40px\\",
    color: colors.grey900
  },
  headline: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"400\\",
    fontSize: \\"24px\\",
    lineHeight: \\"80px\\",
    color: colors.grey900
  },
  subheading2: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"400\\",
    fontSize: \\"16px\\",
    lineHeight: \\"80px\\",
    color: colors.grey900
  },
  subheading1: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"400\\",
    fontSize: \\"16px\\",
    lineHeight: \\"120px\\",
    color: colors.grey900
  },
  body2: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"500\\",
    fontSize: \\"14px\\",
    lineHeight: \\"24px\\",
    color: colors.grey900
  },
  body1: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"400\\",
    fontSize: \\"14px\\",
    lineHeight: \\"20px\\",
    color: colors.grey900
  },
  caption: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"400\\",
    fontSize: \\"12px\\",
    lineHeight: \\"15px\\",
    color: colors.grey900
  },
  button: {
    fontFamily: \\"Arial\\",
    fontWeight: \\"500\\",
    fontSize: \\"14px\\",
    lineHeight: \\"18px\\",
    color: colors.grey900
  }
};

",
  "function _objectSpread(target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i] != null ? arguments[i] : {}; var ownKeys = Object.keys(source); if (typeof Object.getOwnPropertySymbols === 'function') { ownKeys = ownKeys.concat(Object.getOwnPropertySymbols(source).filter(function (sym) { return Object.getOwnPropertyDescriptor(source, sym).enumerable; })); } ownKeys.forEach(function (key) { _defineProperty(target, key, source[key]); }); } return target; }

function _defineProperty(obj, key, value) { if (key in obj) { Object.defineProperty(obj, key, { value: value, enumerable: true, configurable: true, writable: true }); } else { obj[key] = value; } return obj; }

function _typeof(obj) { if (typeof Symbol === \\"function\\" && typeof Symbol.iterator === \\"symbol\\") { _typeof = function _typeof(obj) { return typeof obj; }; } else { _typeof = function _typeof(obj) { return obj && typeof Symbol === \\"function\\" && obj.constructor === Symbol && obj !== Symbol.prototype ? \\"symbol\\" : typeof obj; }; } return _typeof(obj); }

function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError(\\"Cannot call a class as a function\\"); } }

function _defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if (\\"value\\" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } }

function _createClass(Constructor, protoProps, staticProps) { if (protoProps) _defineProperties(Constructor.prototype, protoProps); if (staticProps) _defineProperties(Constructor, staticProps); return Constructor; }

function _possibleConstructorReturn(self, call) { if (call && (_typeof(call) === \\"object\\" || typeof call === \\"function\\")) { return call; } return _assertThisInitialized(self); }

function _assertThisInitialized(self) { if (self === void 0) { throw new ReferenceError(\\"this hasn't been initialised - super() hasn't been called\\"); } return self; }

function _getPrototypeOf(o) { _getPrototypeOf = Object.setPrototypeOf ? Object.getPrototypeOf : function _getPrototypeOf(o) { return o.__proto__ || Object.getPrototypeOf(o); }; return _getPrototypeOf(o); }

function _inherits(subClass, superClass) { if (typeof superClass !== \\"function\\" && superClass !== null) { throw new TypeError(\\"Super expression must either be null or a function\\"); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, writable: true, configurable: true } }); if (superClass) _setPrototypeOf(subClass, superClass); }

function _setPrototypeOf(o, p) { _setPrototypeOf = Object.setPrototypeOf || function _setPrototypeOf(o, p) { o.__proto__ = p; return o; }; return _setPrototypeOf(o, p); }

import React from \\"react\\";
import colors from \\"../colors.js!=!/Users/mathieudutour/Projects/Lona/compiler/webpack-loader/lib/loader.js!../colors.json?__forceLona=1\\";
import shadows from \\"../shadows.js!=!/Users/mathieudutour/Projects/Lona/compiler/webpack-loader/lib/loader.js!../shadows.json?__forceLona=1\\";
import textStyles from \\"../textStyles.js!=!/Users/mathieudutour/Projects/Lona/compiler/webpack-loader/lib/loader.js!../textStyles.json?__forceLona=1\\";

var Button =
/*#__PURE__*/
function (_React$Component) {
  _inherits(Button, _React$Component);

  function Button() {
    _classCallCheck(this, Button);

    return _possibleConstructorReturn(this, _getPrototypeOf(Button).apply(this, arguments));
  }

  _createClass(Button, [{
    key: \\"render\\",
    value: function render() {
      var Text$text;
      var View$backgroundColor;
      var View$hovered;
      var View$onPress;
      var View$pressed;
      View$backgroundColor = colors.blue100;
      Text$text = this.props.label;
      View$onPress = this.props.onTap;

      if (View$hovered) {
        View$backgroundColor = colors.blue200;
      }

      if (View$pressed) {
        View$backgroundColor = colors.blue50;
      }

      if (this.props.secondary) {
        View$backgroundColor = colors.lightblue100;
      }

      return React.createElement(\\"button\\", {
        style: Object.assign({}, styles.view, {
          backgroundColor: View$backgroundColor
        }),
        onClick: View$onPress
      }, React.createElement(\\"span\\", {
        style: styles.text
      }, Text$text));
    }
  }]);

  return Button;
}(React.Component);

export { Button as default };
;
var styles = {
  view: {
    alignItems: \\"flex-start\\",
    backgroundColor: colors.blue100,
    display: \\"flex\\",
    flex: \\"0 0 auto\\",
    flexDirection: \\"column\\",
    justifyContent: \\"flex-start\\",
    paddingTop: \\"12px\\",
    paddingRight: \\"16px\\",
    paddingBottom: \\"12px\\",
    paddingLeft: \\"16px\\"
  },
  text: _objectSpread({
    textAlign: \\"left\\"
  }, textStyles.button, {
    alignItems: \\"flex-start\\",
    display: \\"block\\",
    flex: \\"0 0 auto\\",
    flexDirection: \\"column\\",
    justifyContent: \\"flex-start\\"
  })
};",
]
`;
